### DISCUSSION TITLE
v5.11.2

### DISCUSSION THREAD

**Discussion by MuellerSeb** [mhm-ufz/mHM#36] [Show and tell]
# Release Notes

### Enhancements

- documentation modernized with [doxygen-awesome-css](https://github.com/jothepro/doxygen-awesome-css) (!86)
- cmake update to be able to install mHM (`cmake --install`) (!85)
- added pFUnit tests thanks to Nicola Döring (!76)
- link to a new [YouTube tutorial](https://youtu.be/FGJOcYEzbP4) for compiling mHM with cygwin by Mehmet Cüneyd Demirel added to the documentation (!74)
- NetCDF output: add deflate and precision option to namelists (!73)
- refactor cmake workflow (!72)

### Bugfixes

- fixed: `mrm` tried to write output even if routing was switched off (!82)
- unreachable `else` branch in `feddes_et_reduction` removed (!77)
- unnecessary `inout` variable intent in `soil_moisture` removed (!77)

<hr /><em>This discussion was created from the release <a href='https://github.com/mhm-ufz/mhm/releases/tag/v5.11.2'>v5.11.2</a>.</em>

**Comment by pallavhydro** [comment]
I must say, the documentation (https://mhm.pages.ufz.de/mhm/latest/) is much crispier now! And also thanks for the `dark` and `bright` theme toggle button!
